pkg/config/output.go

```diff
@@ -29,6 +29,7 @@ type Output struct {
 	Color               string
 	PrintIssuedLine     bool   `mapstructure:"print-issued-lines"`
 	PrintLinterName     bool   `mapstructure:"print-linter-name"`
+	PrintSeverity       bool   `mapstructure:"print-severity"`
```

Comment:
I don't really like this `print-severity` field in general, but I want to try to maintain older behaviour if possible. I was thinking of adding something like `exit-non-zero-severity` to replace the default severity field. Still need to look at this codebase more to figure out how that's implemented.